test/e2e/mapml-viewer/staticAttribute.test.js

```diff
@@ -0,0 +1,59 @@
+import { test, expect, chromium } from '@playwright/test';
+
+test.describe("Adding Static Attribute to mapml-viewer", () => {
+  let page;
+  let context;
+  test.beforeAll(async function() {
+    context = await chromium.launchPersistentContext('');
+    page = context.pages().find((page) => page.url() === 'about:blank') || await context.newPage();
+    await page.goto("staticAttribute.html");
+  });
+  test.afterAll(async function () {
+    await context.close();
+  });
+
+  test("Setting New Static Attribute to mapml-viewer", async () => {
+    //setting static attribute in the mapml-viewer tag
+    await page.$eval("body > mapml-viewer", (viewer) => viewer.setAttribute("static",""));
```

Comment:
instead of setAttribute you can try using the setter here to test your static API also. When you use the API to set the static attribute you can also add an `expect(_____.hasAttribute("static")).toEqual(true)` or something of that nature to test to see if the attribute gets set.